src/app/routes/app-routes.tsx

```diff
@@ -82,144 +87,172 @@ export const homePageModalRoutes = (
+              </AccountGate>
+            }
+          />
+          <Route
+            path={RouteUrls.ChooseAccount}
+            element={
+              <AccountGate>
+                <ChooseAccount />
+              </AccountGate>
+            }
+          >
+            {ledgerJwtSigningRoutes}
+          </Route>
 
-        <Route
-          path={RouteUrls.SignIn}
-          element={
-            <OnboardingGate>
-              <SignIn />
-            </OnboardingGate>
-          }
-        />
-        <Route
-          path={RouteUrls.AddNetwork}
-          element={
-            <AccountGate>
-              <AddNetwork />
-            </AccountGate>
-          }
-        />
-        <Route
-          path={RouteUrls.ChooseAccount}
-          element={
-            <AccountGate>
-              <ChooseAccount />
-            </AccountGate>
-          }
-        >
-          {ledgerJwtSigningRoutes}
-        </Route>
+          <Route
+            path={RouteUrls.Fund}
+            element={
+              <AccountGate>
+                <FundPage />
+              </AccountGate>
+            }
+          >
+            <Route path={RouteUrls.ReceiveStx} element={<ReceiveStxModal />} />
+            <Route path={RouteUrls.ReceiveBtc} element={<ReceiveBtcModal />} />
+          </Route>
+          <Route
+            path={RouteUrls.FundChooseCurrency}
+            element={
+              <AccountGate>
+                <ChooseCryptoAssetToFund />
+              </AccountGate>
+            }
+          >
+            <Route path={RouteUrls.ReceiveStx} element={<ReceiveStxModal />} />
+          </Route>
 
-        <Route
-          path={RouteUrls.Fund}
-          element={
-            <AccountGate>
-              <FundPage />
-            </AccountGate>
-          }
-        >
-          <Route path={RouteUrls.ReceiveStx} element={<ReceiveStxModal />} />
-          <Route path={RouteUrls.ReceiveBtc} element={<ReceiveBtcModal />} />
-        </Route>
-        <Route
-          path={RouteUrls.FundChooseCurrency}
-          element={
-            <AccountGate>
-              <ChooseCryptoAssetToFund />
-            </AccountGate>
-          }
-        >
-          <Route path={RouteUrls.ReceiveStx} element={<ReceiveStxModal />} />
-        </Route>
+          {sendCryptoAssetFormRoutes}
 
-        {sendCryptoAssetFormRoutes}
+          <Route
+            path={RouteUrls.ViewSecretKey}
+            element={
+              <AccountGate>
+                <ViewSecretKey />
+              </AccountGate>
+            }
+          />
+          <Route path={RouteUrls.Unlock} element={<Unlock />}>
+            {leatherIntroDialogRoutes}
+          </Route>
+        </Route>
 
         <Route
-          path={RouteUrls.ViewSecretKey}
+          path="approver"
           element={
-            <AccountGate>
-              <ViewSecretKey />
-            </AccountGate>
+            <ContainerLayout
+              header={
+                <Header
```

Comment:
This `<Header` component became overly complex when trying to be a do everything for everyone header. 

In fact, once I had applied logic I reverse engineered from production and documented [here](https://github.com/leather-wallet/extension/issues/4371#issuecomment-1919114939) it was decided that for most popups we would only show the logo anyway. 

Steps I think we can take here are:
- refactor and organise our routes 
- create simpler `Containers` and `Headers`